src/main/java/net/education/kyivstar/config/ConfigDataBase.java

```diff
@@ -0,0 +1,73 @@
+package net.education.kyivstar.config;
+
+import net.education.kyivstar.services.util.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+
+import java.util.Map;
+
+public class ConfigDataBase {
+    private String url;
+    private int port;
+    private String dbName;
+    private String directory;
+    private String user;
+    private String password;
+
+    private static final Logger logger = LoggerFactory.getLogger(ConfigDataBase.class);
+
+    public ConfigDataBase() {
+        loadConfig();
+    }
+
+    public String getUser() {
+        return user;
+    }
+
+    public String getPass() {
+        return password;
+    }
+
+    public String getDirectory() {
+        return directory;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public Integer getPort() {
+        return port;
+    }
+
+    public String getDbName() {
+        return dbName;
+    }
+
+    public void loadConfig() {
+
+        try (var inputStream = Utils.class.getClassLoader().getResourceAsStream("config/application.yml")) {
```

Comment:
add null checks to handle cases where the YAML data or specific keys may be missing